src/components/ebay-carousel/README.md

```diff
@@ -4,25 +4,50 @@
 
 ```marko
 <ebay-carousel>
-    <div>item 1</div>
-    <div>item 2</div>
+    <ebay-carousel-item>Item 1</ebay-carousel-item>
+    <ebay-carousel-item>Item 2</ebay-carousel-item>
+    <ebay-carousel-item>Item 3</ebay-carousel-item>
 </ebay-carousel>
 ```
 
-## ebay-carousel Attributes
+This component will bundle different resources depending on Lasso flags provided by your application.
+Use `touch` for touch devices, and `no-touch` for non-touch devices. Without flags, `no-touch` is assumed.
+Currently, the `touch` carousel with type="continuous" will use native scrolling, which has more limited functionality.
+
+[//]: # (TODO: `touch` should ideally be default, but currently the js behavior doesn't execute with native scrolling)
 
+## ebay-carousel Attributes
 Name | Type | Stateful | Description
 --- | --- | --- | ---
-`class` | String | No | custom class
-`index` | String | Yes | zero-based starting position
 `type` | String | No | "continuous" (default) or "discrete"
-`aria-label-prev` | String | No | aria-label for previous control
-`aria-label-next` | String | No | aria-label for next control
+`accessibility-prev` | String | No | aria-label for previous control (default: "Previous Slide")
+`accessibility-next` | String | No | aria-label for next control (default: "Next Slide")
+
+### Additional Attributes for type="continuous"
+Name | Type | Stateful | Description
+--- | --- | --- | ---
+`index` | String | Yes | 0-based index position
+
+### Additional Attributes for type="discrete"
+Name | Type | Stateful | Description
+--- | --- | --- | ---
+`slide` | String | Yes | 1-based slide position
+`items-per-slide` | String | No | 1 - 5 -- number of items for a single carousel slide (default: 1)
+`accessibility-status` | String | No | status text (default: "Showing Slide {currentSlide} of {totalSlides} - Carousel")
+`accessibility-current` | String | No | pagination current slide text (default: "Current Slide {currentSlide} - Carousel")
+`accessibility-other` | String | No | pagination other slide text (default: "Slide {slide} - Carousel")
 
 ## ebay-carousel Events
+Event | Data | Description
+--- | --- | ---
+`carousel-update` | { [visibleIndexes] } | called whenever item visibility changes, including initialization
+`carousel-translate` | | start of carousel content translation
+`carousel-next` | | click next control
+`carousel-prev` | | click previous control
+
+### Additional Events for type="discrete"
+Event | Data | Description
+--- | --- | ---
+`carousel-slide` | `{ slide }` | new slide is navigated to (by controls, dots, or API)
```

Comment:
Do we need this "slide" event? I think `carousel-prev` and `carousel-next` events give us what we need for tracking clicks on those controls. For the dots, perhaps we have a separate `carousel-pagination-click` event that could be used for tracking...? As far as tracking the API call, I don't think we need to do that, the app could handle that when it makes the call. Maybe I'm missing some other purpose of this event though.

Also wondering if "translate" is a bit of a technical term, and something more like "animate" or "move" might be less ambiguous...? I know translate is used in CSS, but trying to find a word that fit's in more with the carousel metaphor.

Reply:
The main idea here was to have controls-based events (used for tracking) which are distinct from state-based events like movement. That was the main purpose of `translate` -- since the actual movement is closer to how teams would use it, like in lazyloading. I'm ok  with renaming this to `move`.

Having a separate `pagination-click` event would be ideal, but unfortunately that would need a bunch of refactoring. I did put a note in the code: `// TODO: add carousel-dot event and remove simulated dot clicks`. Currently the `slide` event kind of makes up for the lack of a `pagination-click`.

Reply:
Ok thanks. That's clearer now. I'm fine with leaving 'slide' in for now so long as we deprecate it when pagination-click comes along. I also find the word 'slide' a bit confusing, as we are using it as both a noun and verb in our lexicon, I can live with it though (the documentation does clear up any confusion quickly), but if anybody can think of a better name before Monday, let us know!